apps/macos/react-native.config.js

```diff
@@ -15,14 +15,16 @@
  */
 'use strict';
 
+const path = require('path');
+
 const macSwitch = '--use-react-native-macos';
 const windowsSwitch = '--use-react-native-windows';
 
 if (process.argv.includes(macSwitch)) {
   process.argv = process.argv.filter((arg) => arg !== macSwitch);
   process.argv.push('--config=metro.config.macos.js');
   module.exports = {
-    reactNativePath: require.resolve('react-native-macos'),
+    reactNativePath: path.dirname(require.resolve('react-native-macos/package.json')),
   };
 } else if (process.argv.includes(windowsSwitch)) {
   process.argv = process.argv.filter((arg) => arg !== windowsSwitch);
```

Comment:
For curiosity sake: Any idea why we don't need to follow the same pattern for `react-native windows`

Reply:
We should be, but since this is the macOS test app, it doesn't really matter. We should really just remove the `windows` case and clean up the file a bit to avoid confusion. I can do that later after I figure out how to make the bundle step pass on CI.

Reply:
I think this is a success :smile:

<img width="1074" alt="image" src="https://user-images.githubusercontent.com/4123478/87342596-cfef3180-c54b-11ea-9753-0134946d6340.png">

FYI @HeyImChris

Reply:
Gotcha. Yeah we don't need that switch, but if we dont' remove it with this CR, we probably will when we move to 0.62 since we won't need the switched anymore, so I wouldn't worry about that. If you're done with work and need an owner to merge it in, I'll happily do so now.

Reply:
Please go ahead and merge it. Thanks 🙏